app/lib/geo.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# Shared geographic calculations
+module Geo
+  class << self
+    # calculates distance between two coordinates
```

Comment:
This is a good candidate for a yarddoc style documentation to make it clear how this method is expected to be used and what it will return: https://gist.github.com/chetan/1827484

Reply:
Not sure I did proper yarddoc style but added comments.